cockroachcloud/sessions-page.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Sessions Page
+summary: The Sessions page provides details of all open sessions in the cluster.
+toc: true
+---
+
+The **Sessions** page of the CockroachCloud Console provides details of all open sessions in the cluster.
+
+To view this page, click **Sessions** in the left-hand navigation of the CockroachCloud Console.
+
+## Sessions list
+
+Use the **Sessions** list to see the open sessions in the cluster. This includes active and idle sessions.
+
+{{site.data.alerts.callout_info}}
+A session is *active* if it has an open transaction (including implicit transactions, which are individual SQL statements), and *idle* if it has no open transaction. Active sessions consume hardware resources.
+{{site.data.alerts.end}}
+
+- If an active session also has an open transaction, the most recent SQL statement will be displayed in the **Statement** column.
+- To view [details of a session](#session-details), click on the **Session Age**.
+
+<img src="{{ 'images/cockroachcloud/sessions-page.png' | relative_url }}" alt="CockroachCloud Console Database Tables View" style="border:1px solid #eee;max-width:100%" />
+
+The following are displayed for each active session:
+
+Parameter | Description
+--------- | -----------
+Session Age | Amount of time the session has been open.
+Txn Duration | Amount of time the transaction has been active, if there is an open transaction.
+Statement Age | Amount of time the SQL statement has been active, if there is an active statement.
+Memory Usage | Amount of memory currently allocated to this session / maximum amount of memory this session has ever been allocated.
```

Comment:
```suggestion
Memory Usage | Amount of memory currently allocated to this session, followed by the maximum amount of memory this session has ever been allocated.
```

I find this clearer when not studying the screenshot and just reading the table, but not an essential change.